README.md

```diff
@@ -1 +1,41 @@
-# portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+"Portfolio" project
+> Portfolio Project is going to be an open project which began now to the end of training. In this Project
+
+
+## Built With
+
+- HTML
+- CSS
+- Technologies used
+
+## Getting Started
+The link bellow is provided to clone the project from GitHub
+git clone https://github.com/OLIPLICHE/portfolio
+
+Clone this in your local repository
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@OLIPLICHE]
```

Comment:
Great job putting your github handle on your readme.

- [ ] [ Optional ]Here is a way you can improve user experience on all your readmes:

```markdown
![OLIPLICHE's GitHub stats](https://github-readme-stats.vercel.app/api?username=OLIPLICHE&count_private=true&theme=dark&show_icons=true)

👤 **OLIPLICHE's Info**
- GitHub: [@githubhandle](https://github.com/OLIPLICHE)
```
This will give you this :

![OLIPLICHE's GitHub stats](https://github-readme-stats.vercel.app/api?username=OLIPLICHE&count_private=true&theme=dark&show_icons=true)

👤 **OLIPLICHE's Info**
- GitHub: [@githubhandle](https://github.com/OLIPLICHE)